Capacitor show w_ accesory.tsv

```diff
@@ -0,0 +1 @@
Light	Source 4 26�	575W	Marley area	E4	4	R54	271	30	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	17�	17�	12�	14lb	222	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	FE2	False	0"	0"	13'10.224"	10'9.778"	0"	-	1658.1.1.0.0	-
Light	Dyna-Spot	1kw/1.5kw	-	CATWALK RIGHT	1	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	1"	-	23�	17�	14�	12�	74.5lbw	-	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Dyna-Spot	FOH	False	False	-	False	0"	0"	34'.241"	-5'3.33"	0"	-	1659.1.1.0.0	-
Light	Dyna-Spot	1kw/1.5kw	-	CATWALK LEFT	1	-	-	-	-	-	-	-	-	-	-	-	-	-	-	-	1"	-	23�	17�	14�	12�	74.5lbw	-	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Dyna-Spot	FOH	False	False	-	False	0"	0"	-35'4.966"	-5'6.18"	0"	-	1660.1.1.0.0	-
Light	Source 4 26�	575W	Michelle	B1	6	L009	141	220	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	17�	17�	12�	14lb	-	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	FC1	False	0"	0"	-21'7.61"	37'5.483"	0"	-	1661.1.1.0.0	-
Light	Source 4 26�	575W	Marley Area	C1	7	R54	142	7	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	25�	17�	17�	14lb	222	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	FB2	False	0"	0"	-15'0 1/2"	32'0 1/2"	0"	-	1662.1.1.0.0	-
Light	Source 4 26�	575W	Marley Area	B5	2	R54	328	26	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	17�	17�	12�	14lb	222	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	FA2	False	0"	0"	20'4.446"	37'4.472"	0"	-	1663.1.1.0.0	-
Light	Source 4 26�	575W	Marley Area	C5	3	R54	284	27	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	17�	17�	12�	14lb	222	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	FB2	False	0"	0"	20'4.446"	32'.889"	0"	-	1664.1.1.0.0	-
Light	Source 4 36�	575W	walkway dh of W6	CATWALK LEFT	2	L200	101	426	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	34�	17�	23�	12�	14lb	-	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4 36�	FOH	False	False	W6	False	0"	0"	30'11.001"	22'8.73"	216'0"	-	1666.1.1.0.0	-
Light	Source 4 26�	575W	Splat H Blue	CATWALK LEFT	3	R80	98	459	-	-	-	-	-	A	-	-	-	-	-	-	1"	6.25"	25�	17�	17�	12�	14lb	222	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4  26�	FOH	False	False	Splat H	False	0"	0"	30'11.296"	3'8.725"	0"	-	1668.1.1.0.0	-
Light	Source 4 19�	575W	Maggie Special	D4	2	L201	241	207	-	-	-	-	-	A	-	IRIS	-	-	-	-	1"	6.25"	17�	17�	12�	12�	14lb	-	0�	-	0�	0"	-	False	False	False	Focus Point	5'0"	0�	0"	0�	0"	0�	0"	0�	0"	0�	Etc Source 4 19�	FOH	False	False	FC3	False	0"	0"	6'2.094"	19'.447"	0"	-	2208.1.1.0.0	-

```

Comment:
You should dump this file in the `lib` folder or create a new `data` folder so that your top level structure stays clean. http://stackoverflow.com/questions/9889007/rails-where-to-put-data-files